webapi/src/main/scala/org/knora/webapi/messages/v2/responder/ontologymessages/OntologyMessagesV2.scala

```diff
@@ -1818,9 +1818,12 @@ case class PredicateInfoV2(predicateIri: SmartIri, objects: Seq[OntologyLiteralV
    * @return the predicate's IRI object.
    */
   def requireIriObject(errorFun: => Nothing): SmartIri =
+    getIriObject().getOrElse(errorFun)
```

Comment:
Deprecation annotation missing or is this by purpose?

Reply:
TBH, we have so many deprecation warnings and I did not want to add to it.
Problem is that the compile errors are on top of the warnings, so if I have a compile error I have to scroll every time.
I even considered removing the other deprecation annotations.

Reply:
That was actually my guess. Best would be to replace deprecated methods instead of removing just the annotation. But we already have it for a while... I would jump back into removal process once I'm done with current tasks (possibly next week).